app/views/projects/index.html.erb

```diff
@@ -19,8 +20,9 @@
   </colgroup>
   <thead>
     <tr>
-      <th>Id</th>
-      <th>Name</th>
+      <th>Project Id</th>
+      <th></th>
```

Comment:
```suggestion
      <th>Project State</th>
```

Reply:
this column header is deliberately left blank

Reply:
Not blocking are empty table headers good for accessibility?